src/main/java/me/cominixo/signcopy/mixin/AbstractSignBlockMixin.java

```diff
@@ -24,13 +22,13 @@ public class AbstractSignBlockMixin {
 
     @Inject(method = "onUse", at = @At("HEAD"))
     public void onSignUse(BlockState state, World world, BlockPos pos, PlayerEntity player, Hand hand, BlockHitResult hit, CallbackInfoReturnable<ActionResult> cir) {
-        if (player.isSneaking()) {
+        // Lazy approach to comparison, but it works for the moment.
+        if (player.getStackInHand(hand).toString().equals("1 air")) {
```

Comment:
There's no reason to do this, 
```java
if (player.getStackInHand(hand).isEmpty())
```
will work.

Reply:
Fixed.